src/main/java/org/symphonyoss/integration/webhook/jira/parser/v2/JiraMetadataParser.java

```diff
@@ -121,6 +125,21 @@ protected void preProcessInputData(JsonNode input) {
   }
 
   /**
+   * This method parses the issue description to avoid invalid characters
+   * @param input
+   */
+  private void processSummary(JsonNode input) {
+    JsonNode fieldsNode = input.path(ISSUE_PATH).path(FIELDS_PATH);
+    JsonNode summaryNode = fieldsNode.path(SUMMARY_PATH);
+
+    if (summaryNode != null) {
+      SafeString summary = new SafeString(summaryNode.asText());
+      summary.replaceLineBreaks();
```

Comment:
You might use: SafeString summary = ParserUtils.escapeAndAddLineBreaks(summaryNode.asText());